### PR TITLE
Issue #1118 Update docs for registry container

### PR DESCRIPTION
**Pull request by mikehaertl**
@solidnerd I was very courageous with my changes - I hope you don't mind.

What I did:

* Add a section "Configuration" after "Installation" and move the config settings there
* Remove the full `docker-compose.yml` example as it hides the really relevant parts for the registry. We already explain the Gitlab setup in the main README.
* Remove the part about official certificats. As I see it, they are not needed. And if someone wants to use them, they will know what to do.
* Only list the relevant changes in docker-compose.yml and for the Nginx registry setup

If you feel that I changed it too much, let me know and I can change it back.

### CODE REVIEW

File: docs/container_registry.md [hunk 1]
```diff
@@ -63,134 +185,6 @@ gitlab:
...
-    environment:
-    - REGISTRY_LOG_LEVEL=info
-    - REGISTRY_STORAGE_FILESYSTEM_ROOTDIRECTORY=/registry
-    - REGISTRY_AUTH_TOKEN_REALM=http://gitlab.example.com:10080/jwt/auth
-    - REGISTRY_AUTH_TOKEN_SERVICE=container_registry
-    - REGISTRY_AUTH_TOKEN_ISSUER=gitlab-issuer
-    - REGISTRY_AUTH_TOKEN_ROOTCERTBUNDLE=/certs/registry-auth.crt
-    - REGISTRY_STORAGE_DELETE_ENABLED=true
-```
-> **Important Notice**
```
Comment (solidnerd) [inline docs/container_registry.md]: I think we should keep this information

File: docs/container_registry.md [hunk 2]
```diff
@@ -16,7 +16,129 @@ Since `8.8.0` GitLab introduces container registry. GitLab is helping to authent
   - [Docker GitLab](https://github.com/sameersbn/docker-gitlab) >= 8.8.5-1
 
 
-# Available Parameters
+# Installation
+
+## Setup with Nginx as Reverse Proxy
+
+We assume that you already have Nginx installed on your host system and that
+you use a reverse proxy configuration to connect to your Gitlab container.
```
Comment (solidnerd) [inline docs/container_registry.md]: Replace Gitlab with GitLab

File: docs/container_registry.md (hunk 1 above)
Comment (mikehaertl) [reply docs/container_registry.md]: @solidnerd Added it back in.

**Comment by mikehaertl** [comment]
@solidnerd I've fixed Gitlab to GitLab everywhere and also rebased the changes to master to fix a merge conflict due to 10.1.2 upgrade.

Let me know if there's more that should be changed.

**Comment by solidnerd** [comment]
@mikehaertl LGTM !

**Pull request merged**